<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>可视化方式</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="visualComboBox">
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string notr="true"/>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="maxVisibleItems">
           <number>10</number>
          </property>
          <property name="placeholderText">
           <string>未选择</string>
          </property>
          <property name="duplicatesEnabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>起点终点</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>矩形障碍</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>圆形障碍</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="gridSize">
          <property name="value">
           <double>12.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>栅格大小</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>二阶惩罚半径</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>一阶惩罚半径</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="fstPenaltyRad">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="maximum">
           <double>9999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>2.000000000000000</double>
          </property>
          <property name="value">
           <double>24.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QDoubleSpinBox" name="sedPenaltyValue">
          <property name="suffix">
           <string/>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>9999.000000000000000</double>
          </property>
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QDoubleSpinBox" name="fstPenaltyValue">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>9999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>5.000000000000000</double>
          </property>
          <property name="value">
           <double>20.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="sedPenaltyRad">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="maximum">
           <double>9999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>4.000000000000000</double>
          </property>
          <property name="value">
           <double>48.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>一阶惩罚值</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>二阶惩罚值</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>清空地图</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="drawCorridor">
          <property name="text">
           <string>绘制飞行走廊</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="drawOptWaypoint">
          <property name="text">
           <string>绘制轨迹点</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pathPlanButton">
        <property name="text">
         <string>启动路径规划</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="interactive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
